servers/src/main/java/tachyon/master/file/FileSystemMasterClientServiceHandler.java

```diff
@@ -22,6 +22,7 @@
 
 import tachyon.Constants;
 import tachyon.TachyonURI;
+import tachyon.client.file.options.SetAclOptions;
```

Comment:
the server side should not be using client side options ... this is a bad dependency

there should be a separate SetAclOptions in tachyon.master.file.options (similar to CompleteFileOptions for instance) ... I see that SetStateOptions has that problem too and it would be great if you could add a TODO for that